### DISCUSSION TITLE
I saved my workflow in orange and it show error

### DISCUSSION THREAD

**Discussion by Terkaaf** [biolab/orange3#6481] [Q&A]
I mistakenly used the save as option to save my workflow in orange and now I can’t retrieve my data that I took 3 days to prepare. How do I get to previous workflows on orange to retrieve my work

**Comment by janezd** [comment]
This is not a question about Orange but about the operating system and, in particular, the file system. If you have overwritten a file and you don't have any kind of backups, there is nothing you can do. I'm sorry for your loss...
